metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorFastPartitionService.java

```diff
@@ -135,13 +136,13 @@ public HiveConnectorFastPartitionService(
         final HiveConnectorInfoConverter hiveMetacatConverters,
         final ConnectorContext connectorContext,
         final ThreadServiceManager threadServiceManager,
-        final DataSource dataSource,
+        @Qualifier("hiveJdbcTemplate") final JdbcTemplate jdbcTemplate,
```

Comment:
It doesn't look like this class is marked as a `Component` or `Service` so isn't this qualifier kind of useless? It should be where the bean is created (probably a configuration file) on the method input there

Reply:
Agreed, this is redundant. It is already been defined in the config.